tests/test_dbms.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+import time
+
+import pytest
+
+import pixeltable as pxt
+from pixeltable.env import Env
+
+from .utils import reload_catalog
+
+
+class TestDbms:
+    def test_create_table(self):
+        if os.environ.get('PIXELTABLE_DB_CONNECT_STR') is None:
+            logging.error('PIXELTABLE_DB_CONNECT_STR is not set, skipping test')
```

Comment:
No need for this; it isn't necessarily an error. If you use pytest.skip() (see next comment) then you can report all the relevant info that way.